### PR TITLE
Merge 1.2.x

### PR DESCRIPTION
**Pull request by akelad**
**Proposed changes**:
- merge 1.2.x into 1.3.x

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by akelad** [comment]
@ricwo the ordered time stamp things is something you already put in the 1.3.x branch right? So I just rejected the changes for that from this PR. Also i'm pretty sure the io.open change shouldn't be a think here 🤔

**Comment by ricwo** [comment]
yes, the SQL thing is already part of `1.3.x` - can't you just open your SSL PR against 1.3.x?

**Comment by akelad** [comment]
well, i could but then the changelog of 1.2.8 and stuff isn't included. I don't see what the issue of merging this PR is? I would just remove the `io.open` change

File: rasa/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "1.3.3"
+__version__ = "1.3.3"
```
Comment (ricwo) [inline rasa/version.py]:
```suggestion
__version__ = "1.3.3"

```

**Comment by ricwo** [comment]
the `1.2.8` stuff was already released in `1.3.x` as `1.3.2` and would now be duplicated in the changelog. so yeah you could merge this but then also remove that part of the changelog

**Comment by akelad** [comment]
i wouldn't remove it from the changelog tbh, otherwise the 1.2.8 release is empty... and it should be in the changelog shouldn't it?

**Comment by ricwo** [comment]
it's already in the changelog under 1.3.2 (master and 1.3.x), and since we'll merge 1.3.x back into master it doesn't make sense to duplicate it

**Comment by akelad** [comment]
but then do we leave out 1.2.9 completely? That feels really weird

**Comment by ricwo** [comment]
😄 It's fine to duplicate imo, but let's add a label to the `1.2.9` entry (something like `backport of 1.3.2`)

**Pull request merged**